components/fxa-client/Cargo.toml

```diff
@@ -41,6 +39,6 @@ mockiato = { git = "https://github.com/myelin-ai/mockiato.git", rev = "bfd8d83a6
 prost-build = "0.5"
 
 [features]
-browserid = ["openssl"]
+browserid = ["openssl", "rc_crypto/hawk"]
```

Comment:
Seeing this reminded me that I have unfinished business over in https://github.com/mozilla/application-services/pull/1005; I'm going to timebox an attempt to rebase it atop this PR so that we can remove this `openssl` dependency :-)